.github/CODEOWNERS

```diff
@@ -29,22 +29,22 @@
 # directory at the root of the repository and any of its
 # subdirectories.
 #/build/logs/ @doctocat
-/tests/disaster-recovery/regional-dr/ @red-hat-storage/turquoise-squad
+/tests/functional/disaster-recovery/regional-dr/ @red-hat-storage/turquoise-squad
 /tests/lvmo/ @red-hat-storage/aqua-squad
-/tests/manage/z_cluster/ @red-hat-storage/brown-squad
-/tests/manage/pv_services/ @red-hat-storage/green-squad
-/tests/manage/storageclass/ @red-hat-storage/green-squad
-/tests/manage/monitoring/ @red-hat-storage/blue-squad
-/tests/manage/mcg/ @red-hat-storage/red-squad
-/tests/manage/rgw/ @red-hat-storage/red-squad
-/tests/ecosystem/upgrade/ @red-hat-storage/purple-squad
-/tests/e2e/flowtest/ @red-hat-storage/magenta-squad
-/tests/e2e/lifecycle/ @red-hat-storage/magenta-squad
-/tests/e2e/workloads/ @red-hat-storage/magenta-squad
```

Comment:
we are missing `tests/cross_functional/system_test` for magenta i guess. @PrasadDesala wdyt?

Reply:
yes. Please also add
/tests/cross_functional/system_test/
/tests/cross_functional/kcs/

Reply:
Done in https://github.com/red-hat-storage/ocs-ci/pull/9605/commits/f36cdc47e1c5887117a4cab240c40fce6df4de3c